dj_rql/filter_cls.py

```diff
@@ -890,7 +890,11 @@ def _convert_value(cls, django_field, str_value, use_repr=False):
             return float(val)
 
         elif filter_type == FilterTypes.DECIMAL:
-            return round(float(val), django_field.decimal_places)
+            integer_part = len(val.split('.', 1)[0])
+            context = decimal.Context(
+                prec=django_field.decimal_places + integer_part, rounding=decimal.ROUND_DOWN,
```

Comment:
Please add additional prec count logic for 0 integer part.
Input 0.123456789 with 4 decimal places filed should give 0.1234, now it's 0.12345.

Reply:
Thanks, @net-free. Fixed.
My previous solution with "Context" showed unexpected behavior. 'Prec' always skips zero when counting, even if it comes after the dot. Not only when it's a fractional part (for example: 0.012345 -->0.01234, 0.0012345 --> 0.001234)